Tools/nuget/make_zip.py

```diff
@@ -1,4 +1,4 @@
-#! /usr/bin/python3
+#! /usr/bin/tauthon3
```

Comment:
tauthon3 -> tauthon

Kind of surprising this one is /usr/bin/... instead of /usr/bin/...

They should probably fix it in cpython, not sure what it's even doing in the Python 2.7 source.